### DISCUSSION TITLE
MTV Tr3s | (US)

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#783] [Q&A]
### Channel Name

MTV Tr3s

### Channel ID

MTVTr3sEast.us

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
